tests/test_client.py

```diff
@@ -216,77 +216,6 @@ def test_choose_host_raises_exception(self):
             client.choose_host(service)
 
 
-@mock.patch("apiron.client.Timeout")
-@mock.patch("apiron.client.get_adapted_session")
-@mock.patch("apiron.client.build_request_object")
-@mock.patch("requests.adapters.HTTPAdapter", autospec=True)
-@mock.patch("requests.Session", autospec=True)
-class TestClientStubCall:
```

Comment:
Stubs are no longer callable directly with `client.call`, because stubs don't invoke the client any longer. `SomeService.some_stubbed_endpoint()` continues to work as expected.